cmux-tui/scripts/smoke-windows-conpty-resize.py

```
@@ -172,9 +314,31 @@ def main() -> None:
+                    "screen",
+                    "wait",
+                    "--pattern",
+                    unicode_result,
+                    "--timeout-ms",
+                    "10000",
+                )
+                unicode_screen = run_cli(
+                    binary, socket_path, env, "terminal", terminal, "screen", "read"
+                )
+                if unicode_result not in unicode_screen.stdout:
+                    raise AssertionError(unicode_screen.stdout)
```

Ruff (0.16.1)

[warning] Abstract `raise` to an inner function

(TRY301)